cyclops-ui/src/components/pages/History.tsx

```diff
@@ -222,7 +222,9 @@ const ModuleHistory = () => {
       <Modal
         title="Manifest"
         open={manifestModal.open}
-        onCancel={handleCancelManifest}
+        onOk={handleCancelManifest}
+        //onCancel={handleCancelManifest}
```

Comment:
@Wasiim105 can you uncomment this line, so the Modal closes when you click outside of it?